_posts/2024-01-31-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2024.05
+author: Priya
+date: 2024-01-31
+category: Jekyll
+layout: post
+---
+
+### Defect Fixes:
+* Resolved an issue where the detailed error information for the Big Query export was not displaying.
+* Resolved an issue where the Data Flow status was sometimes showing incorrect tasks processed.
+* Third party connector modal :Resolved an issue where identity creation was not allowed when the "Allow this app to use my shared access permissions" checkbox was unchecked on the permission page.
```

Comment:
Zoom connector: Resolved an issue when an identity doesn't get created if the user unchecks an optional permission in the Zoom permission grant page.